### ISSUE TITLE
Revision changes view is cutting off text

### ISSUE THREAD

**Issue by philip** [BookStackApp/BookStack#610]
#### Environment

* BookStack Version: v0.18.5
* PHP Version: 7.1.12
* MySQL Version: 5.7.20
* app-editor: markdown
* Other: local setup via `valet`

##### Current Behavior

The attached screenshot is from `books/foo/page/bar/revisions` and is only showing a few characters from each line.

##### Steps to Reproduce

1. Create a book -> chapter -> page
2. Add content and save
3. Make an edit
4. From the page go to `More` -> `Revisions` and click `Changes` for the edit

##### Potentially related
* I _think_ I created this blank page in `wysiwyg` mode, enabled `markdown`, and then added content and made edits... although I'm not certain nor know if this is related.

![bookstack-issue](https://user-images.githubusercontent.com/42943/33789888-9c9c027e-dc30-11e7-8366-267f5d1820f0.png)

**Comment by AbijeetP** [comment]
> I think I created this blank page in WYSIWYG mode, enabled markdown, and then added content and made edits... although I'm not certain nor know if this is related.

That will cause unintended side-effects. Can you see if you can reproduce this on the demo website here - https://www.bookstackapp.com/#demo

**Comment by philip** [comment]
tl;dr: Maybe this is a doc bug?

I am unable to reproduce using the demo.  There might be several requests here:

- Improve revisioning across different page editors (likely something you're already considering)
- Document the page editor choice in the installation docs
- Document the known limitations in the install/admin guide e.g., 
  - It is an application-wide and not user-specific setting (this surprised me; I only now see this limitation referenced in the user guide but it's not a user setting)
  - Changing midstream could cause problems; another reason the installation guide should reference this important choice
  - When exactly is the `app-editor` default-setting used? Until a choice is saved via the GUI?

I'll play more with BookStack this upcoming weekend, thanks for creating this :)

**Comment by Abijeet** [comment]
@philip - Some of the limitations have been documented here - https://www.bookstackapp.com/docs/user/markdown-editor/

**Comment by philip** [comment]
Thanks @Abijeet, that's a good start. Did you choose the markdown section because wysiwyg is the default? That makes sense, although the topic feels more general here as switching the editor (either way) is what causes the problem.

The installation guide feels like an important place for this information. A "Decide on an editor: wysiwyg or markdown." maybe along with a "Switching editors after adding content can lead to unexpected problems. For details, see X." or some such, where X explains the reasons and that it's a global setting.

**Comment by ssddanbrown** [comment]
I think the fundamental issue for this was found in #1553, due to the similarity in output. Therefore I'll close this off.
